src/components/Dashboard/Analytics/AnalyticsElements.jsx

```diff
@@ -44,7 +406,6 @@ export const Post = styled.div`
     margin-bottom: 10px;
     box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
     color: #eaeaea;
-
     border: 1px solid #151414;
     border-radius: 5px;
```

Comment:
```suggestion
border-radius: 5px;
cursor: pointer;
&:hover {
        transform: scale(1.05);
}
`;
```
add `cursor: pointer;` because the posts should be clickable as well
and the hover effect which you used in other places.